sim/scripts/create_mjcf.py

```diff
@@ -259,6 +286,25 @@ def add_joint_limits(self, root: ET.Element, fixed: bool = False) -> None:
                 upper = str(limits.get("upper", 0.0))
                 joint.set("range", f"{lower} {upper}")
 
+                damping = 0.01
+                keys = stompy.damping().keys()
+                for key in keys:
+                    if key in joint_name:
+                        damping = stompy.damping()[key]
+                joint.set("damping", str(damping))
+
+                stiffness = 0
+                keys = stompy.stiffness().keys()
+                for key in keys:
+                    if key in joint_name:
+                        stiffness = stompy.stiffness()[key]
+
+                joint.set("stiffness", str(stiffness))
+
+                if add_reference_position:
+                    if joint_name in default_standing.keys():
+                        joint.set("ref", str(default_standing[joint_name]))
```

Comment:
We don't use keyframe then?

Reply:
gonna remove this - it seems like adding a reference position doesn't do anything

Reply:
actually i am going to make it so that it manually sets the keypose object based on a hyperparameter and the default positiosn from joints.py